utils/tokens.tsx

```diff
@@ -485,3 +514,7 @@ export function getScaledFactor(amount: number) {
     new BigNumber(amount.toString()).shiftedBy(SCALED_FACTOR_SHIFT).toString()
   )
 }
+
+export function getUnscaledFactor(amount: number) {
```

Comment:
I find this util confusing:
- it seems to be the inverse of `getScaledFactor` but accepts a `number` instead of `BN`
- to be pedantic, `getInverseScaledFactor` seems more accurate. 
- I also don't like utils that say `getFoo` when they don't fetch anything and instead compute, I'd rather say `computeFoo`, but I recognize there's a precedent set by the sister util here.

Reply:
ah yeah, fair callout